Makefile.toml

```diff
@@ -191,6 +191,51 @@ description = "Clean build artifacts"
 command = "cargo"
 args = ["clean", "${@}"]
 
+[tasks.update]
+category = "Dependencies"
+description = "Update dependencies to latest semver-compatible version"
+command = "cargo"
+args = ["update"]
+
+[tasks.outdated]
+category = "Dependencies"
+description = "Look for dependencies with newer, possibly-incompatible versions"
+command = "cargo"
+args = ["outdated", "--workspace", "--root-deps-only"]
+
+[tasks.check-deps]
+category = "Dependencies"
+description = "Check dependencies for lack of use, duplicates, vulnerabilities, etc."
+run_task = [
+    { name = [
+        "unused",
+    ] },
+    { name = [
+        "duplicates",
+    ] },
+    { name = [
+        "audit",
+    ] },
+]
+
+[tasks.unused]
+category = "Dependencies"
+description = "Check for unused dependencies"
+command = "cargo"
+args = ["machete", "--with-metadata"]
```

Comment:
Nice! I was wondering if we should run it as a CI job.

Reply:
Probably a good idea! I've held off for the time being as it can produce some false positives depending on build profile, but we could add it to CI and just see if it ends up being annoying or useful!